ComponentTextKit/TextKit/CKTextKitAttributes.h

```diff
@@ -104,6 +109,7 @@ struct CKTextKitAttributes {
     && maximumNumberOfLines == other.maximumNumberOfLines
     && shadowOpacity == other.shadowOpacity
     && shadowRadius == other.shadowRadius
+    && layoutManagerFactory == other.layoutManagerFactory
```

Comment:
As noted, I moved this earlier in the order of evaluation since it's just a pointer comparison.